Slim/Control/Request.pm

```diff
@@ -2404,7 +2404,13 @@ sub __requestRE {
 
 	for my $names (@$possibleNames) {
 		$regexp .= ',' if $i++;
-		$regexp .= (scalar @$names > 1) ? '(?:' . join('|', @$names) . ')' : $names->[0];
+
+		# Bracket each name using word boundaries to avoid
+		# "play" matching "play", "playlist", and "display".
+
+		my @namelist;
+		push @namelist, '\b' . $_ . '\b' for (@$names);
+		$regexp .= '(?:' . join('|', @namelist) . ')';
```

Comment:
Thanks for this PR! 

Could this be

```perl
$regexp .= '(?:' . join('|', map { "\b$_\b" } @$names) . ')';
```

?

Reply:
@michaelherger you're becoming worse than me :smile:

Reply:
Heh... that's @earlchew's suggestion, not mine 😀.

Reply:
Even the use of map?

Reply:
`map` is great. If that's worse than you, I can live with it 😄.

Reply:
Nothing against map or your one-liner. It was making me smile because you rightfully often pointed out some of Perl code where I condense things way too much and I found this one was not bad 😄